buildSrc/src/main/java/gradlebuild/NativePlatformComponentPlugin.java

```diff
@@ -103,7 +115,8 @@ public void execute(Task task) {
         // This allows using dependency substitution for the root project.
         DependencyHandler dependencies = project.getDependencies();
         project.getDependencies().add("testImplementation", project.getDependencies().project(ImmutableMap.of("path", project.getPath())));
-        dependencies.add("testImplementation", "org.spockframework:spock-core:1.3-groovy-2.5");
+        dependencies.add("testImplementation", "org.spockframework:spock-core:2.3-groovy-3.0");
+        dependencies.add("testImplementation", "junit:junit:4.12");
```

Comment:
JUnit 4.12 is from Dec 04, 2014 and has a CVE, could we at least use JUnit 4.13.2 from 2021?